Svc/ActiveLogger/test/ut/ActiveLoggerTester.cpp

```diff
@@ -150,43 +150,43 @@ TEST(ActiveLoggerTest,FatalTesting) {
 
 }
 
-TEST(ActiveLoggerTest,CircularBufferDump) {
+// TEST(ActiveLoggerTest,CircularBufferDump) {
```

Comment:
@timcanham is there some reason we don't just delete these disabled tests, since the functionality is gone?

Reply:
Removed commented out code.